fulp_modules/features/ert/code/safety_moth.dm

```diff
@@ -0,0 +1,185 @@
+#define SAFETY_MOTH_ENGINEER "Engineering Safety Moth"
+#define SAFETY_MOTH_MEDICAL "Medical Safety Moth"
+#define SAFETY_MOTH_SECURITY "Security Safety Moth"
+
+/// ERT Datum
+/datum/ert/safety_moth
+	mobtype = /mob/living/carbon/human/species/moth
+	leader_role = /datum/antagonist/ert/safety_moth
+	enforce_human = FALSE
+	roles = list(/datum/antagonist/ert/safety_moth)
+	mission = "Ensure that proper safety protocols are being followed by the crew."
+	teamsize = 1
+	polldesc = "an experienced Engineering Safety team"
+	opendoors = FALSE
+/datum/ert/safety_moth/medical
+	leader_role = /datum/antagonist/ert/safety_moth/medical
+	roles = list(/datum/antagonist/ert/safety_moth/medical)
+	polldesc = "an experienced Medical Safety team"
+/datum/ert/safety_moth/security
+	leader_role = /datum/antagonist/ert/safety_moth/security
+	roles = list(/datum/antagonist/ert/safety_moth/security)
+	polldesc = "an experienced Security Safety team"
+
+/// Antagonist status
+/datum/antagonist/ert/safety_moth
+	name = "Engineering Safety Moth"
+	role = "Engineering Safety Moth"
+	outfit = /datum/outfit/centcom/ert/safety_moth
+	var/specialty = SAFETY_MOTH_ENGINEER
```

Comment:
what's the point of this? the defines show that it's the same thing as the name/role of the datum. Also christ the spaces

Reply:
I was originally gonna use it for their equipment like med ert and headset